checker/check_runner.go

```diff
@@ -118,8 +118,11 @@ func MultiCheckAnd(fns ...CheckFn) CheckFn {
 			if minResult.Name == "" {
 				minResult.Name = result.Name
 			}
-			if Bool2int(result.Pass) < Bool2int(minResult.Pass) ||
-				result.Confidence < MaxResultConfidence {
+			if Bool2int(result.Pass) < Bool2int(minResult.Pass) {
```

Comment:
+1 to @laurentsimon comment. Use `MakeAndResult` here, so we don't need to re-write this logic.